app/controllers/api/v1/books_controller.rb

```diff
@@ -10,6 +10,10 @@ def index
       def show
         render json: Book.find(params[:id])
       end
+
+      def open_library
```

Comment:
can we named in more descriptive way? like `search` or something like that

Reply:
Done